js/src/wcdl/section/card/title/index.scss

```diff
@@ -0,0 +1,3 @@
+.wcdl-section-card-title {
+	margin-bottom: calc(var(--large-gap) / 2);
```

Comment:
💅 
The spacing in visual design is 16px, but here is 20px. Looks like their spacing is a bit too wide.

Reply:
I have just changed it to 16px in [`d379b89` (#1264)](https://github.com/woocommerce/google-listings-and-ads/pull/1264/commits/d379b89ab49d917501fc6cd3018c5a1c19347d0f), and also updated the screenshot in the PR description. 

At first I thought of simplifying things and avoiding using more arithmetic acrobatic like `calc(var(--main-gap) / 3 * 2)` (and also to me, 16px and 20px wasn't a huge difference and they look acceptable), but then we already have these in our existing code base. One day we should really clean these up and introduce new easier-to-read SCSS variables for these. 😅

Reply:
> One day we should really clean these up and introduce new easier-to-read SCSS variables for these.

Upvote for this. 💯

Reply:
> One day we should really clean these up and introduce new easier-to-read SCSS variables for these.

I created issue https://github.com/woocommerce/google-listings-and-ads/issues/1429 for this.